integration/build.gradle

```diff
@@ -9,7 +9,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.1.0'
+        classpath 'com.android.tools.build:gradle:2.2.0-beta1'
```

Comment:
Because the build doesn't work otherwise :^) 

See: https://discuss.gradle.org/t/classcastexception-in-gradle-nightly-3-0-20160609/17979